### ISSUE TITLE
Pressing the enter key next to the numpad returns incorrect keyCode

### ISSUE THREAD

**Issue by ndaversa** [blackberry/Ripple-Framework#41]
``` javascript
document.addEventListener("keydown", function (e) {
    console.log (e.keyCode);
});
```

Take the following code and execute it instead QT and press the enter key above shift and then the enter key next to a numeric keypad.

In all other browsers both keys return 13. In QT it returns 0 for the keypad enter.

This bug affects the omnibar in Ripple-UI since pressing the numeric enter will not navigate to a page.

[2052869]

**Comment by ericpearson** [comment]
This issue also happens on Windows

**Comment by jasondscott** [comment]
This issue appears to be fixed in QT 4.8

**Comment by ericpearson** [comment]
Sweet, I love bugs that fix themselves.
